abracadabra/src/writer/hdf5_writer.cc

```diff
@@ -0,0 +1,82 @@
+#include "hdf5_writer.hh"
+
+#include <cstring>
+
+
+hdf5_writer::hdf5_writer():
+    file_(0), irun_(0), iparticle_(0) {}
```

Comment:
This class has 8 members, all of them of type `size_t`, but only 3 of them are initialised.

Reply:
Fixed.